actions/aaa_dbmmods_dependencies_MOD.js

```diff
@@ -1,22 +1,27 @@
 const Mods = {
   DBM: null,
 
-  installModule(moduleName) {
+  async installModule(moduleName, version) {
     return new Promise((resolve) => {
-      require('child_process').execSync(`npm i ${moduleName}`);
+      version
```

Comment:
```js
require('child_process').execSync(`npm i ${moduleName}@${version || 'latest'}`)
```

Reply:
I do not agree with this change, it is not a good idea to be installing @latest of something just because there is not a specific version specified. The module could already be in package.json and just has to be installed.